modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordCSVWriterTest.java

```diff
@@ -96,118 +96,44 @@ public void testWrite() throws Exception {
 	}
 
 	@Test
-	public void testWriteAfterChangeFieldName() throws Exception {
```

Comment:
Hey @linolaoj,

Why did you deleted this test instead of change its expected result?

Thanks.

Reply:
Hi @natocesarrego,
this was accidentally removed on the revert, I'll add it back and update according with the decided behavior.
Thanks!